src/test/java/net/openhft/chronicle/queue/impl/single/AppenderFileHandleLeakTest.java

```diff
@@ -276,11 +278,29 @@ public void tailerShouldOnlyKeepCurrentRollCycleOpen() {
 
     private void assertOnlyCurrentRollCycleIsOpen(long timestamp) {
         BackgroundResourceReleaser.releasePendingResources();
+        /*
+         * "A mapped byte buffer and the file mapping that it represents remain valid until the buffer itself is garbage-collected."
+         *
+         * Given we can't guarantee a GC happens, I wonder if this test can ever not be flaky
```

Comment:
good comment ;) Were you gonna add FlakyTestRunner?

Reply:
Pushed, let's see if it helps